src/_preflights/tw-reset.js

```diff
@@ -82,10 +84,15 @@ Reset links to optimize for opt-in styling instead of opt-out.
 */
 
 a {
-  color: inherit;
+  color: var(--w-color-text-link);
   text-decoration: inherit;
 }
 
+a:hover {
```

Comment:
Why was a:active and a:focus omitted here if we were applying styles from [Fabric's base.css](https://github.com/fabric-ds/css/blob/01fd80a27461cf0d593911e974853f84963d0681/src/base.css#LL53C1-L55C8)?

Reply:
I don't actually remember why? 🤔 I'll check

Reply:
We added everything 😁